yaksh/templates/yaksh/grade_user.html

```diff
@@ -163,29 +163,41 @@
         </div>
     </div>
     <h5>Student answer: </h5>
-    {% for ans in answers %}
-    {% if ans.answer.correct %}
-            <div class="panel panel-success">
-                <div class="panel-heading">Correct answer:
+    {% if question.type == "upload" %}
+    {% if user_file_status %}
+        <a href="{{URL_ROOT}}/exam/manage/download/user_assignment/{{question.id}}/{{data.user.id}}/{{paper.question_paper.quiz.id}}">
+        <div class="panel">
+        Assignment File for {{ data.user.get_full_name.title }}
+        </div>
+        </a>
+    {% else %}
+        <h5>No Assignment submitted by {{ data.user.get_full_name.title }}</h5>
```

Comment:
I guess, the student should also be given a link to download his assignment file in view_answerpaper.html